template-for-recording/record_screen_and_upload.bat

```diff
@@ -32,9 +36,15 @@ echo location of your Java installation.
 goto fail
 
 :findJavaFromJavaHome
-set JAVA_HOME=%JAVA_HOME:"=%
+set "JAVA_HOME=%JAVA_HOME:"=%
```

Comment:
You mentioned this and we also learnt from our users' fix

Reply:
I do not think this is the problem. `JAVA_HOME=%JAVA_HOME:"=%` is a valid substitution syntax.

Reply:
Also notice the `if defined JAVA_HOME goto findJavaFromJavaHome` on line 24. This ensures that the variable is defined.